src/main/java/com/spartronics4915/frc2019/auto/actions/DriveOffHABAndZero.java

```diff
@@ -39,6 +39,20 @@ public boolean isFinished()
     @Override
     public void update()
     {
+        if (!onSlope)
```

Comment:
This can be merged with the below (nested) `if` statement. E.g.:

```java
if (mDrive.getPitch().getDegrees() > kBumpEpsilon && !onSlope)
{
  ...
}
```